hw/scripts/openocd.sh

```diff
@@ -17,6 +17,9 @@
 #
 . $CORE_PATH/hw/scripts/common.sh
 
+CFG_RESET="reset halt"
+GDB=arm-none-eabi-gdb
```

Comment:
Ah, I didn't think of this. gdb has to be platform specific, if we're to use this script to debug other cpu architectures.